Source/com/Communicator.h

```diff
@@ -1152,11 +1154,14 @@ namespace RPC {
 
                     // Anounce the interface as completed
                     string jsonDefaultCategories(Trace::TraceUnit::Instance().Defaults());
+#ifdef WARNING_REPORTING
                     string jsonDefaultWarningCategories(WarningReporting::WarningReportingUnit::Instance().Defaults());
+#endif
                     void* result = _parent.Announce(proxyChannel, message->Parameters());
 
+#ifdef WARNING_REPORTING
                     message->Response().Set(instance_cast<void*>(result), proxyChannel->Extension().Id(), _parent.ProxyStubPath(), jsonDefaultCategories, jsonDefaultWarningCategories);
```

Comment:
I just checked and the unintended forced WARNING_REPORTING dependencies here and above were already fixed in the mean time ( that is the reason the PullRequest has merge conflicts) (and just to make sure this line is not only needed when warning reporting is active so it has been corrected in a different way). 
I propose to just remove this and above changes from the pull requests, that will also solve the merge conflicts.

Reply:
removed this change